text/0123-metrics-correlation.md

```diff
@@ -10,12 +10,17 @@ This RFC addresses the high level metrics to span correlation system.
 # Motivation
 
 We believe the value in a good metrics solution is the correlation to traces and other signals.
-This means that we need to store evidence of metric measurements on the spans.  These
-measurements are automatically added when using the basic `metrics` API.
+This means that we need to store evidence of metrics on the spans in the form of metric summaries.
+These summaries are automatically added when using the basic `metrics` API.
+
+# Terms
+
+* **Connected metric:** a metric that is connected to a trace and not just free floating
+* **Metric summary:** the concept of a summarized metric associated to a span
```

Comment:
For completeness I think it might be worth documenting what measurements are and how we want this to relate in the future.